source/reference/minio-mc/mc-event-add.rst

```diff
@@ -96,38 +95,30 @@ Parameters
 
    Specify multiple value using a comma ``,`` delimiter.
 
-   Defaults to ``put,delete,get``.
+   If note specified, defaults to ``put,delete,get``.
```

Comment:
typo fix
```suggestion
   If not specified, defaults to ``put,delete,get``.
```

Reply:
Random thought: I wonder what happens if you have a space after commas in your string?

Reply:
It treats it as another term in the command and errors out.

Reply:
That's annoying behavior. Maybe call out that whitespace is bad? Because I sure would type it out of habit.